pom.xml

```diff
@@ -18,9 +18,11 @@
         <module>bstats-bukkit</module>
         <module>bstats-bungeecord</module>
         <module>bstats-sponge</module>
+		<module>bstats-velocity</module>
```

Comment:
@kashike

Reply:
He loves my tabs and he knows it. He just hides it well.